docker/.env.sample

```
@@ -2,10 +2,13 @@
 ## Any environment variables supported by InvokeAI can be specified here,
 ## in addition to the examples below.
 
-# INVOKEAI_ROOT is the path to a path on the local filesystem where InvokeAI will store data.
+# HOST_INVOKEAI_ROOT is the path on the docker hosts filesystem where InvokeAI will store data.
 # Outputs will also be stored here by default.
-# This **must** be an absolute path.
-INVOKEAI_ROOT=
+# If relative, it will be relative to the docker directory in which the docker-compose.yml file is located
+HOST_INVOKEAI_ROOT=../../invokeai-data
```

we don't want this to be the default for users who copy .env verbatim
```suggestion
#HOST_INVOKEAI_ROOT=../../invokeai-data
```